openomni/commands.py

```diff
@@ -71,6 +71,7 @@ class PodStatusResponse(_BaseCommand):
 
     def Populate(self):
         self.minutes_active = ((ord(self.data[5]) & 0x0f) << 6) + (ord(self.data[6]) >> 2)
+        self.reservoir_level = round((float((((ord(self.data[6]) & 0x03) << 6) + (ord(self.data[7]) >> 2)))*50)/255, 0)
```

Comment:
The last time this was here, it blew up on some 0x1d responses. It'd be good to add it in a try/except or something.

Reply:
Do you know of any examples of messages that cause it to fail?